html/jamstalldhet.html

```diff
@@ -60,28 +69,37 @@ <h2>Förlängt uppdrag 2016-2018</h2>
           organisationer med liknande verksamhet inom kulturområdet. En skrift
           om jämställdhetsintegrerad bedömning av ansökningar inom stipendier
           och bidrag publicerades 2016.
-          <a href="#"
+          <a href="https://www.konstnarsnamnden.se/default.aspx?id=19113"
             >Jämställdhetsintegrerad bedömning av ansökningar om bidrag och
             stipendier.</a
           >
         </p>
-      </article>
-      
+      </section>
+
       <aside>
         <h3>Mer information</h3>
         <ul>
-          <li><a href="#">Jämställ.nu</a></li>
-          <li><a href="#">Konstnärsnämnden handlingsplan</a></li>
-          <li><a href="#"> Nationella sekreteriatet för genusforskning</a></li>
+          <li><a href="https://www.jamstall.nu/">Jämställ.nu</a></li>
+          <li>
+            <a
+              href="https://www.konstnarsnamnden.se/Sve/Informationssidor/PDFer/JIM_HANDLINGSPLAN_16_9_2013_slutversion.pdf"
+              >Konstnärsnämnden handlingsplan</a
+            >
+          </li>
+          <li>
+            <a href="https://www.gu.se/nsfg"
+              >Nationella sekreteriatet för genusforskning</a
+            >
+          </li>
         </ul>
       </aside>
```

Comment:
This shouldn't be an "aside" I think. It won't match in our design. I would make it an extra paragraph under. What do you think?

Reply:
Since the heading of this list implies "additional information" and the list contains external links I think an aside is ok here. But I agree that it could also be another section with a heading and the list. I don't think either is wrong.